obspy/taup/taup_geo.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Functions to handle geographical points
+
+These functions are used to alow taup models
+to process input data with source and station
+locations given as lstitudes and longitudes. The
+functions are set up to hande an elliptical Earth
+model, but we do not have ellipticity corrections
+for travel times. Although changing the
+shape of the Earth from something other than spherical
+would change the epicentral distance the change
+in distance along the ray path has a larger effect,
+and we do not make that correction.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+import warnings
+
+import numpy as np
+
+from .helper_classes import TimeDistGeo
+
+
+def calc_dist(source_latitude_in_deg, source_longitude_in_deg,
+              station_latitude_in_deg, station_longitude_in_deg,
+              radius_of_earth_in_km, flattening_of_earth):
+    """
+    Given the source and station location, calculate the azimuth and distance
+
+    :param source_latitude_in_deg: Source location latitude in degrees
+    :type source_latitude_in_deg: float
+    :param source_longitude_in_deg: Source location longitue in degrees
+    :type source_longitude_in_deg: float
+    :param station_latitude_in_deg: Station location latitude in degrees
+    :type station_latitude_in_deg: float
+    :param station_longitude_in_deg: Station location longitude in degrees
+    :type station_longitude_in_deg: float
+    :param radius_of_earth_in_km: Radius of the Earth in km
+    :type radius_of_earth_in_km: float
+    :param flattening_of_earth: Flattening of earth (0 for a sphere)
+    :type station_longitude_in_deg: float
+
+    :return: distance_in_deg
+    :rtype: float
+    """
+    try:
```

Comment:
Sorry, I know I recommended to move the import here myself, but looking at it now, I think code would be better readable if you bundle the check for geographiclib at the module imports and set a `HAS_GEOGRAPHICLIB` variable that can be used for if/else clauses inside the routines.

I'll open a PR, that's fastest..